package/system/refpolicy/Makefile

```diff
@@ -0,0 +1,78 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=refpolicy
+PKG_VERSION:=2.20200229
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://github.com/SELinuxProject/refpolicy/releases/download/RELEASE_2_20200229
+PKG_HASH:=dec854512ed00cd057408f330c2cea4de7a4405f7a147458f59c994bf578e4b0
+PKG_INSTALL:=1
+PKG_BUILD_DEPENDS:=checkpolicy/host policycoreutils/host
+
+PKG_MAINTAINER:=Thomas Petazzoni <thomas.petazzoni@bootlin.com>
+
+TAR_OPTIONS:=--transform='s%^refpolicy%$(PKG_NAME)-$(PKG_VERSION)%' -xf -
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/refpolicy
+  SECTION:=system
+  CATEGORY:=Base system
+  TITLE:=SELinux reference policy
+  URL:=http://selinuxproject.org/page/Main_Page
+  DEPENDS:=+@TARGET_ROOTFS_NEEDS_XATTR
```

Comment:
I don't see that symbol `CONFIG_TARGET_ROOTFS_NEEDS_XATTR` defined or used anywhere. Am I missing something?

Reply:
I changed
```
DEPENDS:=+@TARGET_ROOTFS_NEEDS_XATTR
```
to
```
DEPENDS:=@KERNEL_SQUASHFS_XATTR||KERNEL_EXT4_FS_SECURITY||KERNEL_F2FS_FS_SECURITY||KERNEL_UBIFS_FS_SECURITY||KERNEL_JFFS2_FS_SECURITY
```
Please let me know if there is a better solution.